src/Compilers/Core/AnalyzerDriver/AnalyzerExecutor.cs

```diff
@@ -129,13 +138,13 @@ public void ExecuteCompilationEndActions(AnalyzerActions actions)
         /// Executes the compilation end actions.
         /// </summary>
         /// <param name="compilationEndActions">Compilation end actions to be executed.</param>
-        public void ExecuteCompilationEndActions(ImmutableArray<CompilationEndAnalyzerAction> compilationEndActions)
+        public void ExecuteCompilationEndActions(ImmutableArray<CompilationAnalyzerAction> compilationEndActions)
```

Comment:
Should rename to ExecuteCompilationActions (I presume this is common to nested CompilationEndActions and top level CompilationActions)? Also update the doc comments and parameter name accordingly?